src/convertToObject.js

```diff
@@ -7,6 +7,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((entry) => entry.trim())
+    .filter((entry) => entry)
+    .reduce((cssProperties, entry) => {
+      const [key, value] = entry
+        .split(':')
+        .map((spltEntry) => spltEntry.trim());
```

Comment:
It's a minor point, but consider using more descriptive names for the variables inside the `map` call for the split entries, such as `property` and `value`, instead of `spltEntry`. This would increase the readability of your code.